data_pipelines/rne/database/process_rne.py

```diff
@@ -219,30 +219,34 @@ def get_tables_count(db_path):
     return count_pp, count_pm
 
 
-def get_company_data_from_stock(entity):
-    siren = entity.get("siren")
-    date_maj = entity.get("updatedAt")
-    dirigeants = (
-        entity.get("formality", {})
-        .get("content", {})
-        .get("personneMorale", {})
-        .get("composition", {})
-        .get("pouvoirs", [])
+def get_dirigeants_from_entity(entity, type_doc="flux"):
+    if type_doc == "flux":
+        company = entity.get("company", {})
+    elif type_doc == "stock":
+        company = entity
```

Comment:
rather than passing type_doc, why not pass the correct company instead ?

Fewer arg.

you can even use a ternary : 
`company = entity if type_doc == "stock" else company = entity.get("company", {})`

Reply:
Yes, sir